src/ui/pages/Overview/Positions/Positions.tsx

```diff
@@ -545,6 +548,13 @@ function PositionList({
               component: assetPageEnabled ? (
                 <SurfaceItemLink
                   to={`/asset/${position.asset.id}`}
+                  onClick={() => {
+                    walletPort.request('assetClicked', {
+                      assetId: position.asset.id,
+                      pathname,
+                      section: 'Overview',
+                    });
+                  }}
```

Comment:
What if asset link is pressed using the keyboard?

Ideally "screen view" event should be enough for this. Maybe these links should have additional search params to let analytics know that they have been visited from search results.

Reply:
I believe this is an approach for analytics to track all usage of the search page.
However, the previous screen name really should be enough for this. I will ask about this.